rootfs/usr/local/bin/generate-kubecfg

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eo pipefail
+
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+KUBECONFIG_DIR="$(dirname "$KUBECONFIG")"
+EXTRA_ARGS=${@:1}
+GOMPLATE_EXTRA_ARGS=${EXTRA_ARGS:-$GOMPLATE_EXTRA_ARGS}
+
+mkdir -p ${KUBECONFIG_DIR}
+
+function ensure_env_vars() {
+	export KOPS_API_SERVER="${KOPS_API_SERVER:-api.$KOPS_CLUSTER_NAME}"
+	export KOPS_CLUSTER_CA="$(cluster_ca)"
+
+	if [ -z "$KOPS_API_SERVER" ]; then
+		echo "KOPS_API_SERVER not defined"
+		exit 1
+	fi
+
+	if [ -z "$KOPS_CLUSTER_CA" ]; then
+		echo "KOPS_CLUSTER_CA not defined"
+		exit 1
+	fi
+}
+
+function cluster_ca() {
```

Comment:
```suggestion
function get_cluster_ca() {
```

Reply:
I am conflicted. So the function is "generalizable", but the it depends on the `KOPS*` envs, so that's why I think the function should be scoped to `kops`

Reply:
Have made the env vars more general